cdc/sinkv2/ddlsink/cloudstorage/cloud_storage_ddl_sink.go

```diff
@@ -68,18 +68,18 @@ func NewCloudStorageDDLSink(ctx context.Context, sinkURI *url.URL) (*ddlSink, er
 	return d, nil
 }
 
-func (d *ddlSink) generateSchemaPath(def cloudstorage.TableDetail) string {
-	return fmt.Sprintf("%s/%s/%d/schema.json", def.Schema, def.Table, def.Version)
+func (d *ddlSink) generateSchemaPath(def cloudstorage.TableDefinition) string {
```

Comment:
`generateSchemaPath` can be a function, since it does not use any field belongs to the `ddlSink`, and it's stateless.

Reply:
done